_posts/2015-07-09-open-code-of-conduct.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Announcing the Open Code of Conduct
+author: bkeepers
```

Comment:
If there was an easy way to do dual authorship, I'm OK with that otherwise we can leave it as it is, we all worked on this together.